docs/new/references/flashblocks.md

```diff
@@ -0,0 +1,148 @@
+---
+description: Flashblocks support (alpha feature)
+---
+
+New support for "Flashblocks" is now available for alpha testing on Base Mainnet.
+
+ref: https://docs.base.org/base-chain/flashblocks/apps
+
+## Disclaimers
+
+* The only endpoint supporting Flashblocks is `https://base-mainnet-flash.streamingfast.io:443`.
+* That endpoint is not guaranteed to be stable or available at all times.
+* The protocol might still change in the next few weeks, as we gather feedback on usage.
+* Flashblocks coming from Firehose or Substreams endpoint should not be considered final data. Only write to database data coming from full blocks: there is no "undo" mechanism for Flashblocks.
+
+## Description
+
+* Flashblocks are partial blocks that are not fully confirmed yet. 
+* They are emitted every 200ms and contain a fraction of the transactions that will be in the final block.
```

Comment:
Real sentences and not bullets. Let's list also some of the benefits of consuming partial blocks